baip-0002.md

```diff
@@ -0,0 +1,62 @@
+    BAIP: 0002
+    Title: Reform of loopholes in feed price mechanism
+    Authors: cn-vote  bitshareschina@163.com
+    Status: Draft
+    Type: Consensus 
+    Created: 2019-11-5
+
+
+# Abstract
+This BAIP defines reforms to the current feed price mechanism vulnerability. The specific program is: The` feed price` is the highest between the `current price` and the ` two-day moving average price` .
+# Motivation
+After the failure of [BSIP42](https://github.com/bitshares/bsips/blob/master/bsip-0042.md), the current feed price mechanism has major loopholes and serious negatives, many cex exchanges use our vulnerability to maliciously short,which seriously damaged the ecological balance, which caused us to suffer many unnecessary losses and hindered the development of the entire ecology.
+When the vulnerability has been expanded to be intolerable, in an emergency, the passage and execution of [BSIP76](https://github.com/bitshares/bsips/blob/master/bsip-0076.md) has temporarily blocked the expansion of the vulnerability. However, the current feed price mechanism is still in urgent need of reform.
```

Comment:
> in an emergency, the passage and execution of BSIP76

This has the order of events wrong - it was executed prior to achieving any network consensus (both [USD](https://bitsharestalk.org/index.php?topic=29654.msg335815#msg335815) & [CNY](https://bitsharestalk.org/index.php?topic=29653.msg335814#msg335814)) [through devious means](https://bitsharestalk.org/index.php?topic=29659.msg336083#msg336083).

Do you intend for this BAIP to be implemented without waiting for network consensus too?

> temporarily blocked the expansion of the vulnerability

6 weeks (and counting) of fraudulent/fake price feeds doesn't feel temporary at all, especially when the exit conditions rely on the BTS price exceeding the threshold price (unknown/indefinite duration). Further it's failed to follow up with voted upon accurate threshold feed prices - rejecting the reality of the current BTS market rate is a thinly veiled bail out for bad debtors.

Arguably it 'blocked' the alleged risk of CEX short attacks by actively defrauding bitasset holders.

> However, the current feed price mechanism is still in urgent need of reform.

The price feed scripts for select bitassets are in scope for reform, not core/ui based bitshares price feed mechanisms.

Given how BSIP76 is indefinite, there's really no urgency. Your price feed changes won't go live until BSIP76 is dead, so why the rush?